scripts/rest_client.py

```diff
@@ -174,6 +174,15 @@ def show_config(self):
         """
         return self._get("show_config")
 
+    def ping(self):
+        """simple ping"""
+        if not self.show_config():
+            return {"status": "not_running"}
+        elif self.show_config()['state'] == "running":
```

Comment:
we will want to store the result in a variable - this way it'd make the request twice which is not really what we want.

Reply:
I fixed it in the latest commit.